studio/hooks/analytics/useDbQuery.tsx

```diff
@@ -6,28 +7,45 @@ import {
   MetaQueryResponse,
   DbQuery,
 } from 'components/interfaces/Reports/Reports.types'
-import { useStore } from 'hooks'
-import useSWR from 'swr'
+import { useProjectContext } from 'components/layouts/ProjectLayout/ProjectContext'
+import { executeSql } from 'data/sql/execute-sql-query'
 
 type UseDbQuery = (sql: DbQuery['sql'], params?: BaseReportParams) => [DbQueryData, DbQueryHandler]
 const useDbQuery: UseDbQuery = (sql, params = DEFAULT_QUERY_PARAMS) => {
-  const { meta } = useStore()
+  const { project } = useProjectContext()
 
   const resolvedSql = typeof sql === 'function' ? sql(params) : sql
+
   const {
     data,
-    error: swrError,
-    isValidating: isLoading,
-    mutate,
-  } = useSWR<MetaQueryResponse>(resolvedSql, async () => await meta.query(resolvedSql), {
-    revalidateOnFocus: false,
-    revalidateIfStale: false,
-    revalidateOnReconnect: false,
-    dedupingInterval: 2000,
-  })
+    error: rqError,
+    isLoading,
+    isRefetching,
+    refetch,
+  } = useQuery(
+    ['projects', project?.ref, 'db', { ...params, sql: resolvedSql }],
+    ({ signal }) => {
+      return executeSql(
```

Comment:
pg meta queries are not returning any json data as well.

<img width="901" alt="Screenshot 2023-01-25 at 11 46 26 AM" src="https://user-images.githubusercontent.com/22714384/214475472-0b66dd00-85cf-49e5-8123-98fe98a01484.png">

Reply:
Do you have any users in your database? It only returns data for me when there are users (assuming you're on the `reports/auth` page)